docs/user-guide/annotations.md

```diff
@@ -500,3 +503,14 @@ nginx.ingress.kubernetes.io/lua-resty-waf-extra-rules: '[=[ { "access": [ { "act
 ```
 
 For details on how to write WAF rules, please refer to https://github.com/p0pr0ck5/lua-resty-waf.
+
+### Proxy protocol
```

Comment:
If this is specific to SSL passthrough, I'd rather have the feature described in the *SSL Passthrough* section directly.